### PR TITLE
Update topics with grpc-timeout option in header

### PR DESCRIPTION
**Pull request by pwoznic**
Fixes Issue #92

### CODE REVIEW

**Comment by pwoznic** [comment]
Updated several topics with grpc-timeout. I'm also making some improvements to those REST sections. :)

File: www/docs/api-reference/admin-apis/create-corpus.md [hunk 1]
```diff
@@ -1,20 +1,71 @@
...
-This page describes the details of interacting with this endpoint.
+
+### Create Corpus Request Headers
+
+To interact with the Create Corpus service via REST calls, you need the following 
+headers:
+
+* `customer_id` is the customer ID to use for the request.
+* JWT token as your `Bearer Token`
+* (Optional) `grpc-timeout` lets you specify how long to wait for queries that
```
Comment (aamirbutt) [inline www/docs/api-reference/admin-apis/create-corpus.md:26]: wait for the queries -> wait for the call

File: www/docs/api-reference/indexing-apis/file-upload/file_upload.md [hunk 2]
```diff
@@ -84,4 +83,21 @@ $ curl -H "Authorization: Bearer $jwt" -F file=@/tmp/instructions.pdf \\
     -F doc_metadata='{ "filesize\": 1234 }' \\
     'https://${vars['domains.rest.indexing']}:443/v1/upload?c=123456\&o=151'
 `}
+</pre>
+
+## Add a Timeout to the File Upload
+
+Adding `grpc-timeout` to the header of your REST call lets you specify how 
+long to wait for queries that have the potential to take longer to process. We
```
Comment (aamirbutt) [inline www/docs/api-reference/indexing-apis/file-upload/file_upload.md:91]: `wait for the calls` instead of `wait for the queries` is more generic.

**Review by aamirbutt — APPROVED** [review]
Overall looks good. Just one comment about the wording.

**Pull request merged**